packages/backend-ai/src/get_message.test.ts

```diff
@@ -0,0 +1,246 @@
+import { getMessage } from './get_message.js';
+import { ContentBlock, GetMessageInput } from './types.js';
+import { before, describe, it, mock } from 'node:test';
+import assert from 'node:assert';
+import { BedrockRuntimeClient } from '@aws-sdk/client-bedrock-runtime';
+
+void describe('getMessage', async () => {
+  let mockBedrockRuntimeClient: BedrockRuntimeClient;
+
+  before(() => {
+    mockBedrockRuntimeClient = {
+      send: mock.fn(async () => ({
+        output: {
```

Comment:
We should probably type this object as ConverseCommandOutput or whatever the type is called from the SDK